packages/smart-contracts/scripts/test-deploy-batch-conversion-deployment.ts

```diff
@@ -0,0 +1,70 @@
+import '@nomiclabs/hardhat-ethers';
+import { HardhatRuntimeEnvironment } from 'hardhat/types';
+import { deployOne } from './deploy-one';
+
+import { batchConversionPaymentsArtifact } from '../src/lib';
+import { chainlinkConversionPath as chainlinkConvArtifact } from '../src/lib';
+import { CurrencyManager } from '@requestnetwork/currency';
+
+// Deploys, set up the contracts
+export async function deployBatchConversionPayment(
+  args: any,
+  hre: HardhatRuntimeEnvironment,
+): Promise<any> {
+  try {
+    console.log('start BatchConversionPayments');
+    const _ERC20FeeProxyAddress = '0x75c35C980C0d37ef46DF04d31A140b65503c0eEd';
+    const _EthereumFeeProxyAddress = '0x3d49d1eF2adE060a33c6E6Aa213513A7EE9a6241';
+    const _chainlinkConversionPath = '0x4e71920b7330515faf5EA0c690f1aD06a85fB60c';
+    const _paymentErc20ConversionFeeProxy = '0xdE5491f774F0Cb009ABcEA7326342E105dbb1B2E';
+    const _paymentEthConversionFeeProxy = '0x98d9f9e8DEbd4A632682ba207670d2a5ACD3c489';
```

Comment:
It would be better to get these addresses from the previous deployment instead of hardocing them.
If there is an update in the previous script that modify the deployment order it will break the tests.